dali/ft/filecopy.cpp

```diff
@@ -3668,6 +3668,7 @@ cost_type FileSprayer::updateTargetProperties()
                      ((stricmp(aname,"@rowCompressed")==0)&&copyCompressed)||
                      (stricmp(aname,"@local")==0)||
                      (stricmp(aname,"@recordCount")==0)
+                     (stricmp(aname,"@lfnHash")==0)
```

Comment:
Will this prevent lfnhash being set on the target logical file when copying?  Maybe if superTgt is set.

But I am also a bit confused what copying with a target of a superfile actually means.  Do you know?

Reply:
>Will this prevent lfnhash being set on the target logical file when copying? Maybe if superTgt is set.

No, the lfnHash will be calculated and set when the IDistributedFile is attach()'d (which is the point that the logical name is a tied to the instance).